content/en/docs/home/_index.md

```diff
@@ -15,7 +15,7 @@ menu:
     title: "Documentation"
     weight: 10
 description: >
-  Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation.
+  Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation (<a href="https://www.cncf.io/about">CNCF</a>).
```

Comment:
Thanks for your suggestion! Can you please elaborate on the reasoning for this change? In my understanding, we don't need a link in the `description` field: it is used to generate the HTML meta tag (`<meta name=description …>`), with its `content` value in plain text (with no HTML tags).